mne/realtime/fieldtrip_client.py

```diff
@@ -202,6 +209,13 @@ def _guess_measurement_info(self):
                 info._update_redundant()
                 info._check_consistency()
 
+            if chs_unknown:
+                w = ('Following channels in the FieldTrip header were '
+                     'unrecognized and marked as MISC: ')
+                for ch_ in chs_unknown:
+                    w += ch_ + ' '
```

Comment:
We usually don't use single letter variables so ``w`` -> ``msg``. And instead of a for loop I would use ``str.join``. So to reduce 3 lines to one: ``warn(msg + ', '.join(chs_unknown))``.